titanic.ipynb

```diff
@@ -8,6 +8,7 @@
         "## Titanic 3\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 7 to 8_
```python
# Create a countplot of the man/woman/child classification
sns.countplot(data=titanic, x='who', hue='alive', palette='Paired')
```
A new multie line comment made on sns countplot 

_Respond and view the context [here](https://gitnotebooks.com/smith-kyle/notebook-diff-tests/pull/10)._

Reply:
Ok makes snse I'll update this thing